aemedge/blocks/hero/hero.js

```diff
@@ -28,9 +28,13 @@ function decorateMetaInfo() {
   const lastUpdate = getMetadata('modified-time')
     ? getMetadata('modified-time')
     : getMetadata('published-time');
-  infoBlockWrapper.append(
-    span({ class: 'media-blend__date' }, `Updated on ${formatDate(lastUpdate)}`),
-  );
+  if (lastUpdate) {
+    const path = window.location.pathname.split('/');
```

Comment:
`window.location.pathname.indexOf('/news/') > -1` is maybe simpler and can be done in one line

Reply:
I don't want it applied to /some-other-area/news/03/..., just to /news/*

Reply:
Ok than use `window.location.pathname.startsWith('/news/')` still way simpler the splitting and checking the array.

Reply:
Done.